src/sentry/integrations/slack/unfurl/discover.py

```diff
@@ -43,6 +44,13 @@ def unfurl_discover(data, integration, links: List[UnfurlableUrl]) -> UnfurledUr
 
         params = link.args["query"]
         query_id = params.get("id", None)
+        user_id = params.get("user", None)
+
+        projects = list(
```

Comment:
You may be able to avoid doing this passing the user to the internal API request instead, and all this project logic will be handled for you.

Reply:
Oh nice, [updated](https://github.com/getsentry/sentry/pull/27864/files#diff-8d5a6feafdf282ab5d17a07378f22b38d7146702ae739fa6995ff83fce15a915R103)